tasks/uglify.js

```diff
@@ -19,7 +20,7 @@ function relativePath(file1, file2) {
   var file2Dirname = path.dirname(file2);
 
   if (file1Dirname !== file2Dirname) {
-    return path.relative(file1Dirname, file2Dirname) + path.sep;
+    return path.relative(file1Dirname, file2Dirname);
```

Comment:
`relativePath()` is only used once, down here:
https://github.com/gruntjs/grunt-contrib-uglify/pull/472/files#diff-0c8f71b28b3ffffd4d019110b7446b88L139

Since I'm now using `path.join()` instead of string concatentation down below, that trailing (back)slash is no longer required.